### PR TITLE
Create database, Create animals tables

### PR DESCRIPTION
**Pull request by axcell-0**
This pull request includes the following changes :

Created a new database named vet_clinic.
Created a table named animals with the required columns: id, name, date_of_birth, escape_attempts, neutered, weight_kg.
Added SQL code for creating the table in schema.sql.
Inserted provided animal data (Agumon, Gabumon, Pikachu, Devimon) into the animals table.
Added SQL code for data insertion in data.sql.
Wrote SQL queries to fulfill the provided tasks and added them to queries.sql

### screenshot
image
image
image

### CODE REVIEW

File: schema.sql
Comment (Efiamotu-1) [inline schema.sql:1]: - [ ] Your schema is currently incorrect as it is missing two things `) & :`, kindly see the code snippet guide below to correct your error code as currently what you have cannot create an animal table :+1: :sparkles: 
- code snippet
```sql
CREATE TABLE animals (
    id serial PRIMARY KEY,
    name VARCHAR(100),
    date_of_birth DATE,
    escape_attempts INTEGER,
    neutered BOOLEAN,
    weight_kg DECIMAL(5, 2));
```

File: README.md [hunk 1]
```diff
@@ -1 +1,110 @@
...
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Database</summary>
+  <ul>
+	@@ -100,137 +45,39 @@ After you're finished please remove all the comments and instructions!
+
+### Key Features <a name="key-features"></a>
+
+- Database
```
Comment (Efiamotu-1) [inline README.md:47]: - [ ] your readme is currently structured in a way that all of the necessary sections are not displayed at once as shown in the image below unless the arrow beside this highligted text is clicked, please kindly fix this as your readme currently looks like you don't have all of the necessary sections 
rm

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @axcell-0,

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+SELECT name FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
+
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+SELECT * FROM animals WHERE neutered = TRUE;
+
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```
Comment (zeff96) [inline queries.sql]: - [ ] Hello, fantastic job on incorporating queries to address analytical questions! 😊 Upon reviewing the project guidelines, I observed that there's a specific query mentioned that hasn't been addressed in the current project. 🤔 Please double-check to ensure that all the questions outlined in the guidelines are covered by the queries. Thank you! 🚀

Screenshot from 2023-11-20 19-36-56

**Review by zeff96 — CHANGES_REQUESTED** [review]
### Status: CHANGES REQUIRED ♻️
Hi @axcell-0 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost

 ## Highlights
:heavy_check_mark: Correct GitFlow
:heavy_check_mark: Decsriptive PR

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zeff96 — APPROVED** [review]
### Status: APPROVED :green_circle: 
Hi @axcell-0 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congrats

## Highlights
:heavy_check_mark: All queries implemented
:heavy_check_mark: All changes implemented

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**